sky/clouds/service_catalog/__init__.py

```diff
@@ -20,6 +20,9 @@
 def _map_clouds_catalog(clouds: CloudFilter, method_name: str, *args, **kwargs):
     if clouds is None:
         clouds = list(_ALL_CLOUDS)
+        if method_name == "list_accelerators":
+            clouds.append("kubernetes")
+
```

Comment:
This is ok for this PR, but we should add a comment here suggesting to remove this once the common service catalog functions are refactored from `clouds/kubernetes.py` to `kubernetes_catalog.py` ([see todo](https://github.com/skypilot-org/skypilot/blob/1d9c3ec6e5256cae05da352c4a686ffca1d4cb5e/sky/clouds/service_catalog/kubernetes_catalog.py#L17-L18)).

Once that's done, we can safely add `'kubernetes'` to `_ALL_CLOUDS`.

Reply:
Addressed in https://github.com/skypilot-org/skypilot/pull/2638/commits/6b1691e445c3c711a3d93fb5d21da3912d452359